Makefile

```diff
@@ -1,10 +1,12 @@
 .DEFAULT_GOAL := default
 
+RUBY_VERSION := 3.3.4
+
 default: lint test
 lint: buildkite-plugin-lint rubocop shellcheck
 test: buildkite-plugin-test rspec
 
-docker_run := docker run -t --rm -v "${PWD}"/:/plugin:ro -w /plugin
+docker_run := docker run -t --rm -v "${PWD}"/:/plugin -w /plugin
```

Comment:
The `ro` there made the mounted file system read-only which prevent using `bundle exec`. 

I don't know if there was a particular requirement for the file system to be read-only. It was like that from the very first definition, see 0b77225bae7ee6b2a6f15c50d7a942b45ea33c13

As CI shows, using a read-write version does not result in failures, so I think we can continue with it?